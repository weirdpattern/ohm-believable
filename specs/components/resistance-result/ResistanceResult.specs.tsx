import * as React from "react";
import renderer, { ReactTestRendererJSON } from "react-test-renderer";

import ResistanceResult from "../../../src/components/resistance-result";

describe("ResistanceResult components", (): void => {
  it("renders correctly", (): void => {
    const tree: ReactTestRendererJSON = renderer
      .create(
        <ResistanceResult
          a={"yellow"}
          b={"brown"}
          multiplier={"violet"}
          tolerance={"gold"}
        />
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
