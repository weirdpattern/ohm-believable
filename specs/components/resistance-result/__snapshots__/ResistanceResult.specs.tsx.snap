// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ResistanceResult components renders correctly 1`] = `
<div
  className="results"
>
  <h2>
    410 MΩ ±5%
  </h2>
</div>
`;
